.eslintrc.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Remember to start using the `eslintrc.json` from the root of our github folder going forward into other labs.